src/components/Thanks/MyKiva/ThanksBadges.vue

```diff
@@ -167,6 +175,10 @@ const props = defineProps({
 		type: Object,
 		default: () => ({}),
 	},
+	selectedLoan: {
```

Comment:
What is this selected loan?

Reply:
Yeah I didn't see it called out in the requirements, so I didn't worry about having a loan being spotlighted